packages/migrate/Migration.js

```diff
@@ -185,6 +185,14 @@ class Migration {
    * @param  {Object}   options  config and command-line
    */
   async run(options) {
+    // check --migrate-none
+    if (config.migrateNone || config['migrate-none']) {
+        if (config.events) {
+            config.events.emit("migrate:nothingToMigrate");
```

Comment:
I know you copied this from `truffle test --compile-none`, but I think we should use a better message here, e.g. `migrate:skipped`, which would say something like "> Migrations have been skipped, not migrating."

(Maybe there should be an issue to fix this for the compile case)

Reply:
will change accordingly